README.md

```
@@ -157,3 +159,17 @@ enable profiling. Doing this enables the following code-block on the settings fi
 
 See https://www.mediawiki.org/wiki/Manual:$wgProfiler for details
+
+# Deploy on Railway
+
+[![Deploy on Railway](https://railway.com/button.svg)](https://railway.com/deploy/mediawiki?referralCode=hNtFFf&utm_medium=integration&utm_source=template&utm_campaign=generic)
+
+Click the button above to deploy a fully configured Taqasta (MediaWiki) instance on [Railway](https://railway.com/)
+
+The template provisions a Taqasta and required service automatically. During setup you will be prompted to configure the following environment variables. You can go ahead with the defaults and update later or change `MW_ADMIN_USER` and `MW_ADMIN_PASS` before first deploy
+
+### Post-deploy steps
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)